gaphor/adapters/connectors.py

```diff
@@ -197,7 +190,7 @@ def disconnect(self, handle):
 component.provideAdapter(CommentLineElementConnect)
 
 
-@component.adapter(items.DiagramLine, items.CommentLineItem)
+# @component.adapter(items.DiagramLine, items.CommentLineItem)
```

Comment:
Can we remove this commented out adapter?

Reply:
Done.